app/serializers/application_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationSerializer
+  include JSONAPI::Serializer
+  include Lookup
```

Comment:
Do we need `Lookup` in every single serializers? 
If not, we might prefer to only include it in the serializer that will need it.